sound/soc/sof/topology.c

```diff
@@ -2754,7 +2755,16 @@ static int sof_set_dai_config_multi(struct snd_sof_dev *sdev, u32 size,
 				return -ENOMEM;
 
 			/* set cpu_dai_name */
-			dai->cpu_dai_name = link->cpus->dai_name;
+			if (j >= link->num_cpus) {
+				dev_err(sdev->dev,
+					"error: DAI widget count exceeds link cpus count\n");
```

Comment:
can we capitalise "CPUs" too, in fact wouldn't "link CPU count" (without the "s") be more correct - like in the first part "DAI widget count" - not "DAI widgets count?"